Update index.html

kotla-saketh opened the pull request:
Math 4 Python section was updated click on email and phone no. and removed quick access it looks not good. plz don't mind it

kotla-saketh commented:
by clicking the email it will take you to the outlook .it was easy way to contact you 
check out my portifolio in my github for quick access i added hamburger for quick access but unfortunately i have less time so i just stopped the work i think this simple thing is enough to satisfy my pull request.


thank you

codingkatty commented:
Hi there, sorry but I feel like this is not a valid pull request. It has too less changes and the added functions are a little misleading (because it is just a placeholder phone/email). Thanks for the try tho~

kotla-saketh commented:
Ok

On Tue, Oct 22, 2024, 8:16 PM Lim Xin Ying ***@***.***> wrote:

> Hi there, sorry but I feel like this is not a valid pull request. It has
> too less changes and the added functions are a little misleading (because
> it is just a placeholder phone/email). Thanks for the try tho~
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2429493850>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3IHY26KIC2VV75P4KNYCR3Z4ZQOFAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRZGQ4TGOBVGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

kotla-saketh commented:
Normally I seen a word or value in css and html in some repoes are changed
pull requests are also taken .so I thought it may valid it's my first time
to contribute. Ivseen some linkedin repoes and commits.
If possible I want to contribute plz tell any thing you many to change if I
have time I will commit that change.and I seen the logoes in the footer
also linked to the websites I taught directly giving your href links by
your self is right.plz reply.
Most important thing I don't know how you given the hamburger menu it was
not visible to any one it was very small I just sawed now

On Tue, Oct 22, 2024, 8:18 PM Saketh Kotla ***@***.***> wrote:

> Ok
>
> On Tue, Oct 22, 2024, 8:16 PM Lim Xin Ying ***@***.***>
> wrote:
>
>> Hi there, sorry but I feel like this is not a valid pull request. It has
>> too less changes and the added functions are a little misleading (because
>> it is just a placeholder phone/email). Thanks for the try tho~
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2429493850>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A3IHY26KIC2VV75P4KNYCR3Z4ZQOFAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRZGQ4TGOBVGA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

codingkatty commented:
> Normally I seen a word or value in css and html in some repoes are changed pull requests are also taken .so I thought it may valid it's my first time to contribute. Ivseen some linkedin repoes and commits. If possible I want to contribute plz tell any thing you many to change if I have time I will commit that change.and I seen the logoes in the footer also linked to the websites I taught directly giving your href links by your self is right.plz reply. Most important thing I don't know how you given the hamburger menu it was not visible to any one it was very small I just sawed now
> […](#)
> On Tue, Oct 22, 2024, 8:18 PM Saketh Kotla ***@***.***> wrote: Ok On Tue, Oct 22, 2024, 8:16 PM Lim Xin Ying ***@***.***> wrote: > Hi there, sorry but I feel like this is not a valid pull request. It has > too less changes and the added functions are a little misleading (because > it is just a placeholder phone/email). Thanks for the try tho~ > > — > Reply to this email directly, view it on GitHub > <[#122 (comment)](https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2429493850)>, > or unsubscribe > <https://github.com/notifications/unsubscribe-auth/A3IHY26KIC2VV75P4KNYCR3Z4ZQOFAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRZGQ4TGOBVGA> > . > You are receiving this because you authored the thread.Message ID: > ***@***.***> >

Hi, yes typo fixes are usually acceptable. Still, I don't think this should be counted as one as adding the 'phone number' and email, as they are placeholders (fake) and not official.
Here's some ideas for fix: fixing the icon sizes, the css of the 'challenges' page or changing and using a single official font.
For any major changes, its recommended to look into the issues section and create an issue, get assigned and start working on it. Hope you complete hacktoberfest and earn all badges!

kotla-saketh commented:
I fixed icon in the follow us section but as you mentioned about the font
in challenge section I cant find any error in your website related to font

On Wed, Oct 23, 2024, 5:36 AM Lim Xin Ying ***@***.***> wrote:

> Normally I seen a word or value in css and html in some repoes are changed
> pull requests are also taken .so I thought it may valid it's my first time
> to contribute. Ivseen some linkedin repoes and commits. If possible I want
> to contribute plz tell any thing you many to change if I have time I will
> commit that change.and I seen the logoes in the footer also linked to the
> websites I taught directly giving your href links by your self is right.plz
> reply. Most important thing I don't know how you given the hamburger menu
> it was not visible to any one it was very small I just sawed now
> … <#m_-8237210438072275777_>
> On Tue, Oct 22, 2024, 8:18 PM Saketh Kotla *@*.*> wrote: Ok On Tue, Oct
> 22, 2024, 8:16 PM Lim Xin Ying @.*> wrote: > Hi there, sorry but I feel
> like this is not a valid pull request. It has > too less changes and the
> added functions are a little misleading (because > it is just a placeholder
> phone/email). Thanks for the try tho~ > > — > Reply to this email directly,
> view it on GitHub > <#122 (comment)
> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2429493850>>,
> > or unsubscribe >
> https://github.com/notifications/unsubscribe-auth/A3IHY26KIC2VV75P4KNYCR3Z4ZQOFAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRZGQ4TGOBVGA
> > . > You are receiving this because you authored the thread.Message ID: >
> *@*.***> >
>
> Hi, yes typo fixes are usually acceptable. Still, I don't think this
> should be counted as one as adding the 'phone number' and email, as they
> are placeholders (fake) and not official.
> Here's some ideas for fix: fixing the icon sizes, the css of the
> 'challenges' page or changing and using a single official font.
> For any major changes, its recommended to look into the issues section and
> create an issue, get assigned and start working on it. Hope you complete
> hacktoberfest and earn all badges!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2430535682>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3IHY22PYMOZFGAOF2FRDETZ43SAJAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZQGUZTKNRYGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

codingkatty commented:
> I fixed icon in the follow us section but as you mentioned about the font in challenge section I cant find any error in your website related to font
> […](#)
> On Wed, Oct 23, 2024, 5:36 AM Lim Xin Ying ***@***.***> wrote: Normally I seen a word or value in css and html in some repoes are changed pull requests are also taken .so I thought it may valid it's my first time to contribute. Ivseen some linkedin repoes and commits. If possible I want to contribute plz tell any thing you many to change if I have time I will commit that change.and I seen the logoes in the footer also linked to the websites I taught directly giving your href links by your self is right.plz reply. Most important thing I don't know how you given the hamburger menu it was not visible to any one it was very small I just sawed now … <#m_-8237210438072275777_> On Tue, Oct 22, 2024, 8:18 PM Saketh Kotla *@*.*> wrote: Ok On Tue, Oct 22, 2024, 8:16 PM Lim Xin Ying @.*> wrote: > Hi there, sorry but I feel like this is not a valid pull request. It has > too less changes and the added functions are a little misleading (because > it is just a placeholder phone/email). Thanks for the try tho~ > > — > Reply to this email directly, view it on GitHub > <#122 (comment) <[#122 (comment)](https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2429493850)>>, > or unsubscribe > https://github.com/notifications/unsubscribe-auth/A3IHY26KIC2VV75P4KNYCR3Z4ZQOFAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRZGQ4TGOBVGA > . > You are receiving this because you authored the thread.Message ID: > *@*.***> > Hi, yes typo fixes are usually acceptable. Still, I don't think this should be counted as one as adding the 'phone number' and email, as they are placeholders (fake) and not official. Here's some ideas for fix: fixing the icon sizes, the css of the 'challenges' page or changing and using a single official font. For any major changes, its recommended to look into the issues section and create an issue, get assigned and start working on it. Hope you complete hacktoberfest and earn all badges! — Reply to this email directly, view it on GitHub <[#122 (comment)](https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2430535682)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A3IHY22PYMOZFGAOF2FRDETZ43SAJAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZQGUZTKNRYGI> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Sounds great, Imma wait for your pull req!
btw I was mentioning about there were some buttons/lists there that could look better. Once you feel like its enough, you can def send it in! (you do not need to finish all of them)

kotla-saketh commented:
I didn't pull request if you say it is OK just icon correction in follow us
section is enough I will send pull request . If not
If anything I need to change just tell me I will try my best.
I am an student plz understand this is new for me assist me thank you

On Wed, Oct 23, 2024, 8:20 AM Saketh Kotla ***@***.***> wrote:

> I fixed icon in the follow us section but as you mentioned about the font
> in challenge section I cant find any error in your website related to font
>
> On Wed, Oct 23, 2024, 5:36 AM Lim Xin Ying ***@***.***>
> wrote:
>
>> Normally I seen a word or value in css and html in some repoes are
>> changed pull requests are also taken .so I thought it may valid it's my
>> first time to contribute. Ivseen some linkedin repoes and commits. If
>> possible I want to contribute plz tell any thing you many to change if I
>> have time I will commit that change.and I seen the logoes in the footer
>> also linked to the websites I taught directly giving your href links by
>> your self is right.plz reply. Most important thing I don't know how you
>> given the hamburger menu it was not visible to any one it was very small I
>> just sawed now
>> … <#m_-3759116718336764322_m_-8237210438072275777_>
>> On Tue, Oct 22, 2024, 8:18 PM Saketh Kotla *@*.*> wrote: Ok On Tue, Oct
>> 22, 2024, 8:16 PM Lim Xin Ying @.*> wrote: > Hi there, sorry but I feel
>> like this is not a valid pull request. It has > too less changes and the
>> added functions are a little misleading (because > it is just a placeholder
>> phone/email). Thanks for the try tho~ > > — > Reply to this email directly,
>> view it on GitHub > <#122 (comment)
>> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2429493850>>,
>> > or unsubscribe >
>> https://github.com/notifications/unsubscribe-auth/A3IHY26KIC2VV75P4KNYCR3Z4ZQOFAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMRZGQ4TGOBVGA
>> > . > You are receiving this because you authored the thread.Message ID: >
>> *@*.***> >
>>
>> Hi, yes typo fixes are usually acceptable. Still, I don't think this
>> should be counted as one as adding the 'phone number' and email, as they
>> are placeholders (fake) and not official.
>> Here's some ideas for fix: fixing the icon sizes, the css of the
>> 'challenges' page or changing and using a single official font.
>> For any major changes, its recommended to look into the issues section
>> and create an issue, get assigned and start working on it. Hope you
>> complete hacktoberfest and earn all badges!
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2430535682>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A3IHY22PYMOZFGAOF2FRDETZ43SAJAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZQGUZTKNRYGI>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

codingkatty commented:
It's okay. I actually started open source like this month too cuz of hacktoberfest. Take your time!

kotla-saketh commented:
i updated the hamburger now it looks good on the mobile dimension also and
nav top icon shape was also changed to round icon


On Wed, Oct 23, 2024 at 8:28 AM Lim Xin Ying ***@***.***>
wrote:

> It's okay. I actually started open source like this month too cuz of
> hacktoberfest. Take your time!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2430724932>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3IHY22X2HVGH6T2ZUBL4RDZ44GFBAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZQG4ZDIOJTGI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

kotla-saketh commented:
now i am sending pull request in 5 minutes ok plz check after 5 min


On Wed, Oct 23, 2024 at 9:51 AM Saketh Kotla ***@***.***>
wrote:

> i updated the hamburger now it looks good on the mobile dimension also and
> nav top icon shape was also changed to round icon
>
>
> On Wed, Oct 23, 2024 at 8:28 AM Lim Xin Ying ***@***.***>
> wrote:
>
>> It's okay. I actually started open source like this month too cuz of
>> hacktoberfest. Take your time!
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2430724932>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/A3IHY22X2HVGH6T2ZUBL4RDZ44GFBAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZQG4ZDIOJTGI>
>> .
>> You are receiving this because you commented.Message ID:
>> ***@***.***>
>>
>

kotla-saketh commented:
hi pls check


On Wed, Oct 23, 2024 at 9:52 AM Saketh Kotla ***@***.***>
wrote:

> now i am sending pull request in 5 minutes ok plz check after 5 min
>
>
> On Wed, Oct 23, 2024 at 9:51 AM Saketh Kotla ***@***.***>
> wrote:
>
>> i updated the hamburger now it looks good on the mobile dimension also
>> and nav top icon shape was also changed to round icon
>>
>>
>> On Wed, Oct 23, 2024 at 8:28 AM Lim Xin Ying ***@***.***>
>> wrote:
>>
>>> It's okay. I actually started open source like this month too cuz of
>>> hacktoberfest. Take your time!
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2430724932>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/A3IHY22X2HVGH6T2ZUBL4RDZ44GFBAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZQG4ZDIOJTGI>
>>> .
>>> You are receiving this because you commented.Message ID:
>>> ***@***.***>
>>>
>>

kotla-saketh commented:
i want to connect in linkedin some people like you if possible send your
linkedin profile

PR: codingkatty/mathpylearn#122 was not accepted plz consider it again


                 https://www.linkedin.com/in/kotlasaisaketh/

On Wed, Oct 23, 2024 at 10:10 AM Saketh Kotla ***@***.***>
wrote:

> hi pls check
>
>
> On Wed, Oct 23, 2024 at 9:52 AM Saketh Kotla ***@***.***>
> wrote:
>
>> now i am sending pull request in 5 minutes ok plz check after 5 min
>>
>>
>> On Wed, Oct 23, 2024 at 9:51 AM Saketh Kotla ***@***.***>
>> wrote:
>>
>>> i updated the hamburger now it looks good on the mobile dimension also
>>> and nav top icon shape was also changed to round icon
>>>
>>>
>>> On Wed, Oct 23, 2024 at 8:28 AM Lim Xin Ying ***@***.***>
>>> wrote:
>>>
>>>> It's okay. I actually started open source like this month too cuz of
>>>> hacktoberfest. Take your time!
>>>>
>>>> —
>>>> Reply to this email directly, view it on GitHub
>>>> <https://github.com/codingkatty/mathpylearn/pull/122#issuecomment-2430724932>,
>>>> or unsubscribe
>>>> <https://github.com/notifications/unsubscribe-auth/A3IHY22X2HVGH6T2ZUBL4RDZ44GFBAVCNFSM6AAAAABQMUKNP6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZQG4ZDIOJTGI>
>>>> .
>>>> You are receiving this because you commented.Message ID:
>>>> ***@***.***>
>>>>
>>>
